### PR TITLE
Fixes #20320: Split test and build tasks in Jenkins file

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/20320

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: Jenkinsfile [hunk 1]
```diff
@@ -244,16 +180,25 @@ pipeline {
                     }
                 }
                 stage('language') {
-                    agent { 
+                    agent {
                         dockerfile { 
                             filename 'rudder-lang/Dockerfile'
```
Comment (Fdall) [inline Jenkinsfile:185]: Shouldn't it be `language/Dockerfile` ?

Comment (amousset) [reply Jenkinsfile:185]: no, it is named rudder-lang before 7.0

Comment (Fdall) [reply Jenkinsfile:185]: Ok :+1:

**Review by Fdall — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**